### DISCUSSION TITLE
100% postgres data used up without a single query

### DISCUSSION THREAD

**Discussion by bravestonetech** [vercel/community#5535] [Help]
I just enabled the postgres database yesterday for my project and all i did was attempt to connect once and successfully connected. Never ran a single query. I don't even have a table created yet. Today morning, I got an email that 100% of my data transfer usage 256mb has been used up. This definitely seems to be wrong.
Now i can't even try using postgres on my hobby project which would soon have been moved to pro. 

I'm evaluating other hosting options if I won't be able to use postgres. Even if I go for pro, I'm afraid the data transfer limit will hit in 2 days if it's not calculated correctly (without any query by me or my app)

**Comment by bravestonetech** [comment]
I'm temporarily moving to Google cloud as my client wants a working prototype and I'm unable to use vercel for hosting without the database. Will wait for a satisfactory resolution else will move away to Google cloud

**Reply by sherrellbc** [reply]
This happened to me yesterday, actually. I created a new Postgres database instance. I haven't actually finished deploying the app yet the dashboard reports that I am presently at ~70MB of data storage? It can't be correct. The database is _empty_. Maybe it's a bug?

**Comment by amyegan** [comment]
Thanks for reporting this. If there's anything else you can share about your project that could help us investigate the issue, I'll share it with our storage dev team. For example,
- Was the project deployed from a template that uses Postgres?
- What steps did you take to set up the database? Any migrations?
- What is the name of your project or database (to help us location the logs)?

**Reply by bravestonetech** [reply]
* Project doesn't use any postgres template. I had deployed the project earlier. All i did was create a database from the dashboard, then try to connect to it using an external postgres client using the credentials which succeeded.
* Just created a database from the dashboard. Empty. Never ran a single query even to create a table.
* Name of project: `realtormax`. Database: `realtormax-postgres`

**Reply by bravestonetech** [reply]
Due to lack of any response, I've moved my project to google cloud.
